liza/array_values_check.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+def check_values_greater5(input_array)
```

Comment:
`greater_than_five` is more appropriate method name

Reply:
Done